projects/ion/src/lib/badge/badge.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, Input } from '@angular/core';
+import { IconType } from '../icon/icon.component';
+
+export type BadgeType = 'primary' | 'secondary' | 'neutral' | 'negative';
+
+export interface BadgeProps {
+  label?: string;
+  value?: number;
+  icon?: IconType;
+  type: BadgeType;
+}
+
+@Component({
+  selector: 'ion-badge',
+  templateUrl: './badge.component.html',
+  styleUrls: ['./badge.component.scss'],
+})
+export class BadgeComponent {
+  @Input() label?: string;
+  @Input() value?: number;
+  @Input() type!: BadgeType;
+
+  public valueInBadge: string;
+
+  private exists(value: number): boolean {
+    return value !== undefined && value !== null && !isNaN(value);
```

Comment:
```suggestion
    return value !== null && !isNaN(value);
```
`isNaN` already validates undefined values. (more about it [here](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/isNaN))